Src/WebApplication/wwwroot/clientapp/project/create/form.jsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+
+class Form extends React.Component {
```

Comment:
- Validations
- Do not post form, send ajax request.
- Handle errors
- Handle loader